lib/dfe_signin.rb

```diff
@@ -0,0 +1,21 @@
+module DfeSignin
```

Comment:
How about if the filename and case of this namespace matched the underscores used elsewhere, i.e. `dfe_sign_in.rb` and `DfeSignIn`?

Reply:
Yeah, that makes sense, I was treating Signin as one word, but DfE Sign-In is hyphonated, so probably makes sense to camel case them as seperate words